src/Components/Publishing/RelatedArticles/RelatedArticleFigure.tsx

```diff
@@ -39,6 +43,11 @@ export class RelatedArticleFigure extends React.Component<RelatedArticleFigurePr
     const { article } = this.props
     const href = getArticleHref(article.slug)
     const imageSrc = crop(article.thumbnail_image, { width: 510, height: 340 })
+    const bylineArticle = {
+      ...article,
+      layout: "series" as ArticleLayout,
```

Comment:
Why do we manually set layout here?

Reply:
Because the `RelatedArticleFigure` doesn't provide one but is passed in as an `article` anyway. This could also be remedied by making `layout` optional on `article` or adding `layout` to the `RelatedArticleFigure`.

Reply:
Ah I see, yeah I'd prefer making layout optional then, because the query for RelatedArticleFigure on Force looks like [this](https://github.com/artsy/force/blob/master/src/desktop/apps/article/queries/relatedArticles.js) and we'd have to make sure those queries are updated if we wanted `layout` to work for RelatedArticleFigure. Probably not the most ideal solution, but once we get to a place where Reaction Publishing components don't need to pass down everything ie using a store/context then things will feel better.

Reply:
Sounds gut!